documentation/modules/security/proc-installing-certs-per-listener.adoc

```diff
@@ -3,21 +3,41 @@
 // assembly-security.adoc
 
 [id='proc-installing-certs-per-listener-{context}']
-= Providing your own Kafka listener certificates
+= Providing your own Kafka listener certificates for TLS authentication
 
-This procedure shows how to configure a listener to use your own private key and server certificate, called a xref:kafka-listener-certificates-{context}[Kafka listener certificate].
+[role="_abstract"]
+Listeners provide client access to Kafka brokers.
+You configure listeners in the `Kafka` resource, including the configuration required for client access using TLS authentication.
```

Comment:
I would not mention the authentication here => that will again create confusion that the custom certificate is used for client authentication.

```suggestion
You configure listeners in the `Kafka` resource, including the configuration required for client access using TLS.
```